src/App.tsx

```diff
@@ -1,19 +1,17 @@
 import React, { Component } from "react";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 
+// tslint:disable: ordered-imports
 import MainPage from "./components/mainPage/mainPage";
 import Navigation from "./components/navigationBar/navigationBar";
 import ProofEditor from "./components/proofEditor/proofEditor";
+import Einstellungen from "./components/einstellungen/einstellungen";
```

Comment:
Internally in our programfiles the variables and imports should be consistently in english in my opinion.
Even if the displayed text later is in german.

Reply:
Agreed. Though I would like to see the link reflect the component name.